README.md

```diff
@@ -28,6 +28,7 @@ Environment vairables are passed to the `run` command for configuring a mongo-ex
 	--------------------------------|-----------------|------------
 	ME_CONFIG_BASICAUTH_USERNAME    | ''              | mongo-express web username
 	ME_CONFIG_BASICAUTH_PASSWORD    | ''              | mongo-express web password
+	ME_CONFIG_CONNECT_RETRIES       | 5               | Number of startup connection retry attempts to be made
```

Comment:
The rest of these come from `mongo-express` itself directly, right?  I think it would probably be appropriate to check with those upstream maintainers before we synthesize a new one that only applies to this Docker image (esp. to make sure they're OK with both the idea in general, and the chosen environment variable name).

Reply:
Will do, thank you!

Reply:
I read through the upstream project and its open issues, and I agree that it would be better to avoid potential namespace confusion.

Could we call this something a bit different, like `DOCKER_CONNECT_RETRIES`?